app/main.development.js

```diff
@@ -52,8 +53,8 @@ app.on('ready', async () => {
   mainWindow.loadURL(`file://${__dirname}/app.html`);
 
   mainWindow.webContents.on('did-finish-load', () => {
-    mainWindow.show();
-    mainWindow.focus();
+    mainWindow.show(); // eslint-disable-line
```

Comment:
The solution would be to add code that ensures the value of ```mainWindow``` is not null, something like:

```

  mainWindow.webContents.on('did-finish-load', () => {
    if (mainWindow === null) { mainWindow = { show: () => {}, focus: () => {} } } 
    mainWindow.show(); 
    mainWindow.focus(); 
  });
```